src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/EditKmlGroundOverlay/EditKmlGroundOverlay.xaml

```diff
@@ -0,0 +1,24 @@
+﻿<UserControl
+    x:Class="ArcGISRuntime.WPF.Samples.EditKmlGroundOverlay.EditKmlGroundOverlay"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013">
+    <Grid>
+        <esri:SceneView x:Name="MySceneView" />
+        <Border Style="{StaticResource BorderStyle}">
+            <Grid>
```

Comment:
I'd consider putting the slider and label on the same line. I'd also consider showing the current value (basically copying what's done in the map rotation sample)